### PR TITLE
Add barebones fullscreen map

### PR DESCRIPTION
**Pull request by chowington**
Fixes https://github.com/VEuPathDB/web-eda/issues/1307

### CODE REVIEW

File: src/lib/workspace/WorkspaceRouter.tsx [hunk 1]
```diff
@@ -211,6 +212,28 @@ export function WorkspaceRouter({
               </WorkspaceContainer>
             )}
           />
+          <Route
+            path={[
+              `${path}/:studyId/:analysisId/fullscreen-map`,
+              `${path}/:studyId/new/fullscreen-map`,
+            ]}
```
Comment (bobular) [inline src/lib/workspace/WorkspaceRouter.tsx:219]: Could this multiple path matching be used to DRY up the regular analysis routes? (`:analysisId` and `new`)

**Review by bobular — CHANGES_REQUESTED** [review]
This is great, but we need to figure out where to merge it to. It can't be `main` unless somehow the "open fullscreen map" button is easily hidden.

We could just keep a branch going to merge FSM tickets into, and keep merging from main to make the final merge back to main relatively pain free?

**Comment by dmfalke** [comment]
> This is great, but we need to figure out where to merge it to. It can't be `main` unless somehow the "open fullscreen map" button is easily hidden.
> 
> We could just keep a branch going to merge FSM tickets into, and keep merging from main to make the final merge back to main relatively pain free?

This is a good use case for a feature flag. We don't have specific infrastructure for feature flags, but it wouldn't be too hard to add something.

File: src/lib/mapveu/MapVEuFullscreen.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      style={{
+        position: 'fixed',
+        top: 0,
+        left: 0,
+        width: '100vw',
+        height: '100vh',
+        backgroundColor: 'white',
+      }}
+      onKeyUp={(event: React.KeyboardEvent<HTMLDivElement>) => {
+        if (event.key === 'Escape') history.goBack();
```
[the remark this answers is not in the document]
Comment (dmfalke) [reply src/lib/mapveu/MapVEuFullscreen.tsx:25]: It's likely that your focus is still on the button that opened the map. Maybe when the map opens, we can change the focus to the map div?

**Comment by chowington** [comment]
Closing in favor of https://github.com/VEuPathDB/web-eda/pull/1349

**Pull request closed without merging**